CHANGELOG.md

```diff
@@ -12,6 +12,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ### Added
 
+-**CUMULUS-3075**
+  - Changed the API endpoint return value for a granule with no files. When a granule has no files, the return value beforehand for
+    the translatePostgresGranuletoApiGranule, the function which does the translation of a Postgres granule to an API granule, was 
+    undefined, now changed to an empty array. 
+  - Existing behavior which relied on the pre-disposed undefined value was changed to instead accept the empty array.
```

Comment:
There's a section for changes `### Changed` and I think these two points above would be more appropriate there 😮

Reply:
Hey @Nnaga1! To answer your question here:
`Lastly, would I need to write an integration test or not, I'm not sure what the criteria is for when or when not to do so`, I think generally a good rule is integration tests are appropriate for testing that the changes you made work end-to-end. So if you wanted to, you could add a test to https://github.com/nasa/cumulus/blob/master/example/spec/parallel/testAPI/granuleSpec.js to test that calling the granule API endpoint for a modified granule with an empty files objects returns what you expect.